OpenRA.Game/World.cs

```diff
@@ -387,6 +387,7 @@ public void Dispose()
 
 			frameEndActions.Clear();
 
+			Sound.StopAudio();
```

Comment:
The fix for that is easy. Just increase the `exitDelay` in `IngameMenuLogic` and `LeaveMapWidget` to 1400 milisecs.